Subscriber/Cronjob/ImportPaymentMethodSubscriber.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdyenPayment\Subscriber\Cronjob;
+
+use AdyenPayment\Import\PaymentMethodImporterInterface;
+use AdyenPayment\Models\Event;
+use Enlight\Event\SubscriberInterface;
+
+final class ImportPaymentMethodSubscriber implements SubscriberInterface
+{
+    /**
+     * @var PaymentMethodImporterInterface
+     */
+    private $paymentMethodImporter;
+
+    public function __construct(PaymentMethodImporterInterface $paymentMethodImporter)
+    {
+        $this->paymentMethodImporter = $paymentMethodImporter;
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            Event::cronImportPaymentMethods()->getName()=> '__invoke',
+        ];
+    }
+
+    public function __invoke(\Shopware_Components_Cron_CronJob $job)
+    {
+        iterator_to_array(($this->paymentMethodImporter)());
```

Comment:
What should `$this->paymentMethodImporter` return here? The interface and the implementations have importAll and importForShop, but no magic method.

Which action is this event triggering?

Reply:
This was a left over of a refactor that was not yet complete,
this is already corrected and will be available in an update of this PR.